new-lamassu-admin/src/components/tables/DataTable.js

```diff
@@ -86,18 +70,22 @@ const ExpRow = ({
   )
 }
 
-/* rows = [{ columns = [{ name, value, className, textAlign, size }], details, className, error, errorMessage }]
- * Don't forget to include the size of the last (expand button) column!
- */
-const ExpTable = ({
+const DataTable = ({
   elements = [],
   data = [],
   Details,
   className,
+  expandable,
   ...props
 }) => {
   const [expanded, setExpanded] = useState(null)
 
+  const coreWidth = R.compose(R.sum, R.map(R.prop('width')))(elements)
+  const expWidth = 1200 - coreWidth
```

Comment:
noice, so now we don't have to manually reserve extra space for the expand column